File: baekjoon_all/02000+/boj_2562.cpp
// Solve 2022-06-03
// Update 2024-01-28

#include <iostream>
using namespace std;

int main() {
    int max_num = 0;
    int max_idx = 0;

    for (int i = 1; i <= 9; i++) {
        int num;
        cin >> num;

        if (num > max_num) {
            max_num = num;
            max_idx = i;
        }
    }

    cout << max_num << '\n';
    cout << max_idx << '\n';

    return 0;
}

File: baekjoon_all/02000+/boj_2562.py
# Solve 2022-05-19
# Update 2023-10-10

import sys

input = lambda : sys.stdin.readline().rstrip()

max_num = max_idx = 0

for i in range(1, 10):
    num = int(input())

    if num > max_num:
        max_num = num
        max_idx = i

print(max_num, max_idx, sep="\n")
